app/backend/app.py

```diff
@@ -397,6 +407,129 @@ async def list_uploaded(auth_claims: dict[str, Any]):
     return jsonify(files), 200
 
 
+@bp.post("/chat_history")
```

Comment:
Do you think the chat history endpoints could be in a different file/blueprint? That would make it to bring in additional providers (I've been asked about Azure SQL, for example) without bloating app.py.
I know we only have a single file/blueprint for routes now though, so it's possible that I haven't organized it in such a way that it's feasible.

Reply:
Certainly, `app.py` is getting too large, so I think it's good to separate the blueprint. I'll try to restructure it to separate it.

Reply:
Configure separate blueprint in 44bfc40